ntoskrnl/mm/ARM3/sysldr.c

```diff
@@ -3249,7 +3260,21 @@ MmLoadSystemImage(IN PUNICODE_STRING FileName,
     /* FIXME: Call driver verifier's loader function */
 
     /* Write-protect the system image */
+#if !defined(CORE_16449_IS_FIXED) && DBG
+    if (wcscmp(LdrEntry->BaseDllName.Buffer, L"ftfd.dll"  ) == 0 ||
+        wcscmp(LdrEntry->BaseDllName.Buffer, L"win32k.sys") == 0)
```

Comment:
This is a terrible hack, but at least use the unicode functions?

Reply:
(`RtlEqualUnicodeString` or the likes)

Reply:
> unicode functions

That seems needlessly complicated here to me, though I could, if you confirm.